source/extensions/filters/http/aws_lambda/aws_lambda_filter.h

```diff
@@ -13,6 +13,7 @@
 #include "source/extensions/filters/http/aws_lambda/sts_credentials_provider.h"
 
 #include "api/envoy/config/filter/http/aws_lambda/v2/aws_lambda.pb.validate.h"
+#include "nlohmann/json.hpp"
```

Comment:
is this necessary?

Reply:
Removed this line here: fdf3fa903711b4ad10416c0b7939e1d225295033